pom.xml

```diff
@@ -33,6 +33,10 @@
 
         <checkstyle.version>2.17</checkstyle.version>
         <checkstyle.methodNameFormat>^_?[a-z][a-zA-Z0-9]*$</checkstyle.methodNameFormat>
+        <!-- whether autorelease maven central staging repositories - default true to allow review and manually release repositories -->
```

Comment:
Please remove these, not related to the issue at hand and will cause issues with the coming release process changes.